citadel-internal-service-types/src/lib.rs

```diff
@@ -553,6 +555,7 @@ pub struct FileTransferRequestNotification {
     pub cid: u64,
     pub peer_cid: u64,
     pub metadata: VirtualObjectMetadata,
+    pub request_id: Option<Uuid>,
```

Comment:
It looks like there are a few occurrences in tests that didn't get updated to reflect the addition of request_id